unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotComponentView.cs

```diff
@@ -51,9 +55,9 @@ public void SetIsHidden(bool isHidden, string hiddenBy)
             model.isHidden = isHidden;
             model.hiddenBy = hiddenBy;
             hiddenSlot.SetActive(isHidden);
-
             if (isHidden)
             {
+                nftContainer.DOShakePosition(SHAKE_ANIMATION_TIME, 4);
```

Comment:
Can we extract all the animations logic into separated methods? In this way we make the code cleaner and understandable. Wdyt?